LimerickStreetArt.Web/Controllers/HomeController.cs

```diff
@@ -12,17 +12,13 @@
 	public class HomeController : Controller
 	{
 		private readonly StreetArtRepository _streetArtRepository;
-		//private readonly ILogger<HomeController> _logger;
-		//ILogger<HomeController> logger,
-		public HomeController(IConfiguration configuration)
+
+		public HomeController(StreetArtRepository _streetArtRepository)
```

Comment:
remove the underscore from the variable name i.e.

`public HomeController(StreetArtRepository streetArtRepository)`